nest/03.advanced_concepts.md

```diff
@@ -0,0 +1,70 @@
+# 3 Advanced concepts
+In this chapter, we will detail some advanced concepts related to `nest`, which will help understanding how `nest` works, but aren't strictly needed for a daily use of `nest`.
+On the other hand, readers interested in `finest` should read this chapter, because those concepts are important to understand how `finest` works.
+
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+The official repositories are the ones maintained by the Raven-OS team.
+For now, only one is available: `stable`, which is the one enabled by default when installing Nest.
+It contains all the stable and safe-to-use packages for Nest, and is maintained by the Raven-OS team.
+
+### 3.1.2 The unofficial repositories
+There are also unofficial repositories for Nest.
+They are not being maintained by the Raven-OS team or someone trusted by the team.
```

Comment:
```suggestion
These repositories are not vetted by Raven-OS, so caution is advised.
```
This sentence implies a strong negative, when in reality such a repo could have a relation with someone from Raven-OS.